### PR TITLE
Improve README for testing purposes

### PR DESCRIPTION
**Pull request by livanov93**
This partially addresses #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,27 @@
...
+Follow the [instructions](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver#setting-up-a-ur-robot-for-ur_robot_driver) in the paragraph 
+[`Prepare the robot` ](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver#prepare-the-robot)
+
 ## Build Instructions
 
-To build this package in a ROS Foxy workspace, clone this repo
-into the `src/` directory, then:
+To build this package follow the [instructions](https://index.ros.org/doc/ros2/Installation/Foxy/Linux-Install-Debians/) for installation of ROS2.
+
+After installation create a ROS Foxy workspace:
```
Comment (destogl) [inline README.md:14]: Maybe better link [official docs](https://index.ros.org/doc/ros2/Tutorials/Workspace/Creating-A-Workspace/) for this general stuff

File: README.md [hunk 2]
```diff
@@ -18,3 +35,43 @@ rosdep install -y --rosdistro $ROS_DISTRO --ignore-src --from-paths src
...
+```
+ros2 control load_start_controller joint_state_controller
+```
+
+Start the `forward_command_controller`:
+
+```
+ros2 control load_start_controller forward_command_controller_position
+```
+## TODO Run a test node (TBD)
```
Comment (destogl) [inline README.md:62]:
```suggestion

## TODO Run a test node (TBD)
```

File: README.md [hunk 3]
```diff
@@ -18,3 +35,43 @@ rosdep install -y --rosdistro $ROS_DISTRO --ignore-src --from-paths src
...
+Run a test node which will publish joint commands on /forward_command_controller_position/commands (std_msgs::msg::Float64MultiArray)
```
Comment (destogl) [inline README.md:63]: Do you have this test node?

I wrote one in the ros2_control_demos. it is currently under PR ros-controls/ros2_control_demos#53

**Review by destogl — APPROVED** [review]
Just small comments.

File: README.md (hunk 3 above)
Comment (livanov93) [reply README.md:63]: Not yet, but will be added. For now I think it is enough to be able to publish position command. So the other parties can start testing.

File: README.md (hunk 1 above)
Comment (livanov93) [reply README.md:14]: https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/33/commits/e1b1db0928450f67e7159aa3f4dbc79004015795

**Pull request merged**